app/models/admin/account_action.rb

```diff
@@ -73,6 +73,14 @@ def types_for_account(account)
       end
     end
 
+    def disabled_types_for_account(account)
+      if account.suspended?
+        %w(silence suspend)
+      elsif account.silenced?
+        %w(silence)
+      end
```

Comment:
Arguably disable and sensitive don't really make sense for accounts already suspended either?

Reply:
Arguably, yes, but also, I suppose you may want to e.g. suspend the account, then apply one of those restrictions, and finally roll back that the suspend.

Reply:
Maybe, but then would you just freeze?